pages/en/lb4/Booting-an-Application.md

```diff
@@ -0,0 +1,178 @@
+---
+lang: en
+title: 'Booting an Application'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Booting-an-Application.html
+summary:
+---
+
+## What does it mean to boot an Application?
+
+A typical LoopBack application is made up of many artifacts in different files,
+organized in different folders. **Booting an Application** means:
+
+* Discover artifacts automatically based on a convention (a specific folder
+  containing files with a given suffix)
+* Process those artifacts automatically (this usually means binding them to the Application)
```

Comment:
I think we should use fit in the concept of `context` in here somewhere, just to make sure that users understand that `boot` takes care of automatic **`context`** bindings.